templates/repo/diff/box.tmpl

```diff
@@ -1,20 +1,9 @@
-{{if .DiffNotAvailable}}
-	<div>
-		<div class="diff-detail-box diff-box sticky">
-			<div class="ui right">
-				{{template "repo/diff/whitespace_dropdown" .}}
-				{{template "repo/diff/options_dropdown" .}}
-				{{if and .PageIsPullFiles $.SignedUserID (not .IsArchived)}}
-					{{template "repo/diff/new_review" .}}
-				{{end}}
-			</div>
-		</div>
-	</div>
-	<h4>{{.locale.Tr "repo.diff.data_not_available"}}</h4>
-{{else}}
-	<div>
-		<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-ac gt-fw">
-			<div class="gt-df gt-ac gt-fw">
+<div>
+	<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-ac gt-fw">
+		<div class="gt-df gt-ac gt-fw">
+			{{if .DiffNotAvailable}}
+				<h4>{{.locale.Tr "repo.diff.data_not_available"}}</h4>
+			{{else}}
```

Comment:
I think this message should be rendered outside of `.diff-detail-box` like before, e.g. in the place where the normal diff body would be rendered.

Reply:
i've just pushed a new version where the h4 is in diff body.